terraform/transform_destroy_cbd.go

```diff
@@ -134,25 +134,16 @@ type CBDEdgeTransformer struct {
 	// obtain schema information from providers and provisioners so we can
 	// properly resolve implicit dependencies.
 	Schemas *Schemas
-
-	// If the operation is a simple destroy, no transformation is done.
-	Destroy bool
```

Comment:
This was the only remaining thing that was different between `terraform plan -destroy` and `terraform destroy`, right? If so, awesome!